### DISCUSSION TITLE
Disabeling Features

### DISCUSSION THREAD

**Discussion by Hutch79** [mealie-recipes/mealie#3080] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Features like the Meal Planner or the Shopping List are great, but not everyone needs them.

For non-tech savvy peoples, the feature richness of Mealie can be overwhelming.

### Please provide a concise description of the feature that would resolve your issue.

The Feature I would like to request, is to globally disable for example the Timeline, Meal Planner or Shopping Lists.
This way, the UI as well as the dropdowns on the recipes would be less overwhelming.

### Please consider and list out some caveats or tradeoffs made in your design decision

I imagine that hiding the menu elements could be fairly easy, which would already be an improvement.
But to make every button connected to these features toggable could be quit time-consuming.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
I'm generally on board with _something like this_, for the record.

Some considerations would be:
- is it enough to just hide them in the ui?
- does that mess with page layouts? 
- are there issues when using the api related to a disabled feature? Maybe the api operates identically to today 
- is this a group setting or global mealie setting (via env var for instance)

**Comment by SandraCHC** [comment]
For me it is a very impotent feature. Mealie is great but very chaotic with to much option for a small solition with one user

I hope you can add options to disable (with GUI or config parameters) api options, timeline, rating and other things for a slim version
Thank you

**Comment by lscholte** [comment]
I am also looking for ways to reduce the exposed features in mealie. For example, features like timeline, meal planner, and shopping lists are just creating unnecessary clutter for me because I do not use them. Similarly, recipe ratings (mentioned in https://github.com/mealie-recipes/mealie/discussions/2237) is another thing I'd like to be able to remove.
